scalableminds/webknossos
# Better unit scale for dataset scale

This PR reworks the formatScale function that is used to display the scale of a dataset. The function now supports lower resolutions and not only the nano-meter unit.

It also moves the format functions in file dataset_info_tab_view to the dedicated format lib file to have a better structure.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I think this needs to be tested locally
- open a view that displays the scale of a dataset (like the dataset info tab in a tracing)
- check for it to still have the same displayed scale as before
- now decrease the scale drastically up to the next levels. Use a scale like [25000, 25000, 25000] and [25000000, 25000000, 25000000].You can set the scale in the `datasource-properties.json` of the dataset.
- reload the dataset in the dataset view and open a tracing. The displayed scale should be adjusted to micro- and millimeter.


### Issues:
- fixes  #3848

------
